partcad/src/partcad/assembly_factory_assy.py

```
@@ -772,30 +619,19 @@
                         pc_logging.debug("Trying to match ports by name")
                         # Both interfaces are known but they have more than one port each.
                         # Let's find a matching pair of ports.
-                        source_ports = sorted(
-                            list(source_iface_instance.keys())
-                        )
-                        target_ports = sorted(
-                            list(target_iface_instance.keys())
-                        )
+                        source_ports = sorted(list(source_iface_instance.keys()))
```

Ruff (0.8.2)

Unnecessary `list()` call within `sorted()`

Remove the inner `list()` call

(C414)